nomad/fsm.go

```diff
@@ -1833,6 +1835,23 @@ func (s *nomadSnapshot) persistACLTokens(sink raft.SnapshotSink,
 	return nil
 }
 
+func (n *nomadFSM) applySchedulerConfigUpdate(buf []byte, index uint64) interface{} {
```

Comment:
Move to be above here: https://github.com/hashicorp/nomad/blob/4db7e87a31b842ace0c12081aa18229da376cffc/nomad/fsm.go#L998